app/templates/components/code-snippet.hbs

```diff
@@ -1 +1,5 @@
-{{source}}
+<code class="hljs">
+{{#if lineNumbers}}
+{{{source}}}
+{{else}}{{source}}{{/if}}
```

Comment:
To go with the previous comment about didInsertElement: let's remove this conditional and just always do the things that it does when lineNumbers=true.